<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capital Market Line</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 16pt &quot;Segoe UI&quot;;

width: 70%;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 italic 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Capital Market Line</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rp: es el rendimiento esperado de la cartera de inversiones</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Rf: Tasa de retorno libre de riesgo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>E(Rm): Rendimiento esperado del mercado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>σm: Desviación estándar de los rendimientos del mercado, que representa el riesgo total del mercado.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>σp: Desviación estándar de los rendimientos de la cartera, que representa el riesgo de la cartera.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Ingresar valores porcentuales en decimales ejemplo 10% 0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>El combo box Dígitos tiene por default 2 dígitos despues del punto si quieres mas dígitos selecciona las opciones disponibles y presiona otra vez el botón calcular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_rf">
      <property name="placeholderText">
       <string>Rf</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_erm">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>E(Rm)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_des_m">
      <property name="placeholderText">
       <string>σm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_des_p">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>σp</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_calcular">
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_digitos">
        <item>
         <property name="text">
          <string>Digitos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Todos</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_limpiar">
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_generar_pdf">
        <property name="text">
         <string>Generar PDF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextBrowser" name="tb_resultado"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widgetGrafico" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
